formulas/wick-base/functions/wick-make-user

```diff
@@ -4,6 +4,8 @@
 # $1            - Username to create on the system
 # --daemon      - Optional; flag to indicate reasonable settings should be
 #                 assigned for a daemon account.  See the examples.
+# --groups      - Optional; a group (or comma separated list of groups) to 
+#                 associate with this user.
```

Comment:
I would also like for this option to indicate that "The list of groups is added to the user instead of forcing the list of groups to exactly match what's passed in. When this is used, the user's initial group is still automatically created. See `useradd` or `usermod` for further information." Or something similar. The point is that we really want to be explicit about if something is assigning an exact value (such as the home directory) or adding to the list and not unassigning values.